docker-compose.yml

```diff
@@ -14,7 +14,7 @@ networks:
 
 services:
   pmm-server:
-    image: ${PMM_SERVER_IMAGE:-perconalab/pmm-server:dev-latest}
+    image: ${PMM_SERVER_IMAGE:-perconalab/pmm-server-fb:PR-2296-eded043}
```

Comment:
any reason for using this FB?

Reply:
missed this. Restored. Thank you for noticing!